### DISCUSSION TITLE
Dashboard - new view.

### DISCUSSION THREAD

**Discussion by themistervac** [emsesp/EMS-ESP32#1709] [Ideas]
Hello everyone.

In February this year, I assembled the interface and launched it.
I control a Junkers Cerapur Smart ZSB 24-5C gas boiler.

After some time, I came up with some solutions that would make it easier to observe or control selected parameters.

On the main screen after logging in, we have a view of the detected devices. e.g. gas boiler and thermostat etc.

![image](https://github.com/emsesp/EMS-ESP32/assets/11832652/41bbab59-7dd5-4806-92ee-51ac6571da74)

If we click on devices, we see a lot of parameters. Often the interesting ones are at the bottom of the list.

![image](https://github.com/emsesp/EMS-ESP32/assets/11832652/7a1e2ce6-a9d2-4135-8245-def060963047)

It would be nice to be able to choose among these parameters those that we want to be visible immediately after logging in.
Just like in Home assistant or Domoticz.

![image](https://github.com/emsesp/EMS-ESP32/assets/11832652/4b649774-0504-4d61-83f4-5db4c15cd6d5)

Eventually it might look like this 

![image](https://github.com/emsesp/EMS-ESP32/assets/11832652/ab1ef0ac-22d4-4406-a38b-24d859f70100)

or in a simplified form as a list of selected parameters. I know there is entity filtering on the device, but they are not directly on the home screen.

![image](https://github.com/emsesp/EMS-ESP32/assets/11832652/455aae6d-080d-4c8e-8230-8d82d04c9edc)

**Comment by proddy** [comment]
We have the favorite option exactly for this, although its per device. Pick the ones you want to show using Customize Entitites and then filter them using the Favourites 'star' icon.

Also upgrade to the latest version which gives some small UI changes.

**Reply by themistervac** [reply]
Thanks for answer.

Yes, I know that. 
However, it is not displayed directly on the screen after logging in.

<img width="505" alt="image" src="https://github.com/emsesp/EMS-ESP32/assets/11832652/d047585a-6b99-4c8e-9053-faf99427f80a">

I am currently using version 3.6.5 (ESP32)

BTW - great job !

PS
There is a problem with setting the temperature on the thermostat when using the phone - Android 13.
You cannot enter a temperature, e.g. 20.5.

**Comment by proddy** [comment]
what do you mean it's not correctly shown after logging in? The screen shot above shows the 'starred' favourites.

The issue with Android and entering float values could be an issue, I'll need to check. Which locale/language is your Android phone using?

**Comment by themistervac** [comment]
Android 13 MIUI 14.0.4 localisation Poland

I just checked entering the temperature as 19.5.
I see an improvement because I can enter the value after the decimal point.
The problem is the lack of change in the value in the Temperature entity in comfort mode. The value does not change. Maybe because I turned off the heating in the boiler.
